home-manager/install.nix

```
@@ -20,6 +20,12 @@ runCommand "home-manager-install" {
       # Let Home Manager install and manage itself.
       programs.home-manager.enable = true;
 
+      # The host operating system that Home Manager will be installed
+      # on. This enables OS-specific workarounds and tweaks to increase
+      # the general cohesiveness of the system.
+      # See `man home-configuration.nix` for possible values.
```

I know I wrote this, but I'm kinda lukewarm on "possible values."

Certainly not a blocker, but I wonder if anybody has any ideas for better wording.

---

What is bothering you with this wording?

---

I really can't put my finger on it. It just sounds *weird* to me.

Just now, I've found wording I'm satisfied with:
```suggestion
      # For a list of supported systems, see `man home-configuration.nix`.
```

Maybe `s/systems/values/`; I'm fine with either way. It's up to you.